pkg/sdkserver/credentials.go

```diff
@@ -0,0 +1,176 @@
+package sdkserver
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/gptscript-ai/gptscript/pkg/config"
+	gcontext "github.com/gptscript-ai/gptscript/pkg/context"
+	"github.com/gptscript-ai/gptscript/pkg/credentials"
+	"github.com/gptscript-ai/gptscript/pkg/repos/runtimes"
+)
+
+func (s *server) initializeCredentialStore(ctx string) (credentials.CredentialStore, error) {
+	cfg, err := config.ReadCLIConfig(s.gptscriptOpts.OpenAI.ConfigFile)
+	if err != nil {
+		return nil, fmt.Errorf("failed to read CLI config: %w", err)
+	}
+
+	// TODO - are we sure we want to always use runtimes.Default here?
+	store, err := credentials.NewStore(cfg, runtimes.Default(s.gptscriptOpts.Cache.CacheDir), ctx, s.gptscriptOpts.Cache.CacheDir)
```

Comment:
Right now, `runtimes.Default` is the only RuntimeManager in GPTScript, as far as I am aware. I would initialize this the normal way through `s.gptscriptOpts`, but the Runner opts there are nil, so I just did this instead.